test/manual/src/manualTests.ts

```diff
@@ -74,29 +80,40 @@ async function main() {
     .then(() => {
       console.log("signAndCollect successful");
     })
-    .catch(console.log);
+    .catch(e => {
+      console.log("signAndCollect failed");
+      errors.push(e);
+    });
 
   await delay(DELAY_BETWEEN_REQUETS);
 
+  console.log("starting to test sign and cancel");
   try {
-    console.log("starting to test /sign and /cancel");
-
     const response = await client.sign({
       endUserIp: ip,
       personalNumber: personalNumber,
       userVisibleData:
-        "this is a cancellation test - please DO NOT fill in your verification code",
+        "this is a cancellation test - please DO NOT fill in your verification code or cancel the sign from your device",
     });
 
     await cancelOrderIn(client, response.orderRef, 5000);
+    console.log("sign and cancel successful");
   } catch (e) {
-    console.log(e);
+    console.log("sign and cancel failed");
+    errors.push(e);
   }
+
+  return errors;
 }
 
-main()
-  .then(() => console.log("✅ All tests completed successfully"))
-  .catch(err => console.log(err));
+main().then(errors => {
+  const failedTests = errors.length;
+  if (failedTests === 0) {
+    console.log("✅ All tests completed successfully");
+  } else {
+    console.error(`❌ ${failedTests} test(s) failed`);
+  }
+});
```

Comment:
Food for thought - should we keep a `.catch` here just in case? (Skimmed through the test code and I believe the only thing that can throw is client instantiation or `delay` 🤔 )

Reply:
Yes, that is a fair point. Committed your suggestion!